ci_support/run_docker_build.sh

```diff
@@ -40,12 +40,9 @@ echo "$config" > ~/.condarc
 conda clean --lock
 
 conda update --yes --all
-conda install --yes conda-build==1.18.2
+conda install --yes conda-build
 conda info
 
-# Temporary fix to install opengl drivers
-yum install -y mesa-libGL-devel
```

Comment:
We can't merge this PR because of this. I'm going to re-render this one by hand.